examples/dns/setup.js

```diff
@@ -9,15 +9,13 @@ const EXPORTER = process.env.EXPORTER || '';
 
 function setupTracerAndExporters(service) {
   const tracer = new NodeTracer({
-      plugins: {
-          dns: {
-            enabled: true,
-            path: '@opentelemetry/plugin-dns',
-            // Avoid dns lookup loop with http zipkin calls
-            ignoreHostnames: ['localhost']
-        }
-      }
-  });
+    plugins: {
+      dns: {
+        enabled: true,
+        path: '@opentelemetry/plugin-dns',
```

Comment:
why removing:

```
            // Avoid dns lookup loop with http zipkin calls
            ignoreHostnames: ['localhost']
```
because it will cause a loop if it uses Zipkin exporter...

Reply:
Good catch. should have been moved to the options object